config/config.go

```diff
@@ -31,6 +33,9 @@ type IngressConfig struct {
 
 // Get returns an initialized IngressConfig
 func Get() *IngressConfig {
+
+	Hostname, _ := os.Hostname()
```

Comment:
If this fails what happens?
You can expect there to be a `HOSTNAME` env var when running in kubernetes.

Reply:
welp! let's use that then